source/contribute/maintainer.rst

```diff
@@ -57,3 +57,29 @@ the ``use-next-minor-as-default-branch`` switch for the
     $ git switch --create 2.0.x 2.0.0
     Switched to a new branch "2.0.x"
     $ git push doctrine 2.0.x
+
+Tagging a release
+-----------------
+
+We use ``laminas/automatic-releases`` to make releasing as easy as
+closing a milestone. Before closing a milestone, one should check that
+it is complete. If not, they should unassign issues and pull requests,
+or close them / merge them as appropriate. It is also a good idea to
+check which pull requests were merged since the last release on the
+relevant branch, and to make sure each one is properly labelled and
+assigned to the milestone.
+
+Labels are used to generate the release notes, and we should strive to
+provide a consistent experience accross repositories. Here are labels
+that we recommend adding on every repository:
+
+Documentation
+  Includes changes to rst files, markdown files as well as code comments
+  that are not phpdoc comments.
+
+Deprecation
+  For pull requests that *introduce* a new deprecation.
```

Comment:
IMO we should find another label for pull requests that address a deprecation.

Reply:
By "addressing a deprecation", I mean making the deprecation from another package go away, not dropping a legacy code path.

Reply:
I see. In this case, isn't it just an internal change? It shouldn't affect the consumers in any way.

Reply:
Agree, that would be internal. And that what I want to emphasize it, we shouldn't label such a PR as `Deprecation`.

Reply:
By emphasizing "introduce", you imply that there's something else that this label is not about, but I would have never thought of that being a deprecation in another library. The most natural opposite of "introduce" is "fix" but we've already covered it.

Reply:
I added a clarification.